spec/controllers/pages_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Pages', type: :request do
```

Comment:
1. It is not a request
2. 
```suggestion
RSpec.describe 'Pages' do
```
because of: `config.infer_spec_type_from_file_location!`